build.zig

```diff
@@ -55,6 +67,8 @@ pub fn build(b: *std.Build) !void {
 }
 
 pub fn link(b: *std.Build, step: *std.build.CompileStep) void {
+    // TODO - how to get the build option here?
```

Comment:
The way to do this would be a third argument `options: LinkOptions` struct with a `backend: Backend` field in it. However, it's not needed since we can make this a runtime option on the relevant platforms soon.